contracts/interfaces/external/IBiswapRouter02.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
+
+interface IBiswapRouter02 is IUniswapV2Router02 {
```

Comment:
Since `IBiswapRouter02 is IUniswapV2Router02`:

I believe that the functions `getAmountOut` and `getAmountIn` are overloaded, not overridden. This would allow us to use `IBiswapRouter02` instead of `IUniswapV2Router02` in the uniswapLPVaultOperator and allow factorization of the two contracts, for better legibility and maintenance

The biswapLPVaultOperator would use the same code as uniswapLpvault but override the only two functions whose implementation is different.